README.md

```diff
@@ -150,7 +150,48 @@ target:
 `subpath` in the previous url is optional in case your charts are not stored directly under your projects.
 It is worth mentioning that you can use Harbor robot accounts using OCI registries as source or target.
 
-Also, take into account that if you use OCI as the source repository you must specify the list of chart to synchronize
+Also, take into account that if you use OCI as the source repository you must specify the list of charts to synchronize
+or a pointer to an [index file](#oci-index) in the repository.
+
+#### OCI index
+
+In order to speed up the sync process for OCI charts, it is possible to specify a reference to an OCI index file containing 
+the chart versions to be synced from the repository. Thanks to the index, chart-syncer don't need to autodiscover 
+what tags for a specific asset are charts or other kind of artifacts which is a costly process.
```

Comment:
Yeah, I'd not mention this in the Readme, you are explaining implementation details.

In this section I'd just say that you make chart-syncer auto-discover what Helm Charts to sync by providing the URI of a custom index in the OCI registry.